### ISSUE TITLE
Extra filter being added

### ISSUE THREAD

**Issue by paulst1976** [3ll3d00d/beqdesigner#324]
I load a project, select load BEQ filter, once added, highlight one of the filters and select edit (under filter - default) It then adds a 14hz LS / Q 0.7 / Gain 0 to the existing filters..

![Screenshot (18)](https://user-images.githubusercontent.com/45668510/69043631-51a5b400-09eb-11ea-9905-7ead241f4c9b.png)
![Screenshot (19)](https://user-images.githubusercontent.com/45668510/69043654-5bc7b280-09eb-11ea-86cb-c00cf75595d2.png)

**Comment by 3ll3d00d** [comment]
I can't reproduce this

pls provide literally step by step instructions to reproduce as

> I load a project, select load BEQ filter, once added

leaves a lot to the imagination so I might be doing something different

step by step means every single operation you perform to get to this point

**Comment by paulst1976** [comment]
It was happening almost every time at first but just reinstalled the beta3 and I tried it about 10 times since your reply and it only did it once, seems like it may have been a glitch. Now it seems too random to be able to reproduce, I'll try and get a video captured of the specific steps but there's not more to add than what I added already. Don't worry about it, if it happens again I'll try and capture it.. Thanks.
